include/boost/math/quadrature/detail/sinh_sinh_detail.hpp

```diff
@@ -50,7 +50,7 @@ class sinh_sinh_detail
     auto integrate(const F f, Real tolerance, Real* error, Real* L1, std::size_t* levels)->decltype(std::declval<F>()(std::declval<Real>())) const;
 
 private:
-private:
```

Comment:
Why remove it? The `get_abscissa_row` function below is not part of the public API.

Reply:
Because private is literally above it. It is a duplicate